packages/core-mobile/app/vmModule/ModuleManager.ts

```diff
@@ -174,7 +174,12 @@ class ModuleManager {
         ? BlockchainId._2JVSBOINJ9C2J33VNTVZ_YT_VJNZD_N2NKIWW_KJCUM_HUWEB5DB_BRM
         : BlockchainId._2O_YMBNV4E_NHYQK2FJJ_V5N_VQLDBTM_NJZQ5S3QS3LO6FTN_C6FBY_M
     }
-    return BlockchainId._11111111111111111111111111111111LPO_YY
+    if (vmName === NetworkVMType.PVM)
+      return BlockchainId._11111111111111111111111111111111LPO_YY
```

Comment:
so P chain only has 1 blockchain id?

Reply:
It seems like that; however, it generates a different hashed chain ID since the hashBlockchainId function appends a ‘fuji’ prefix when isTestnet is true.